config/routes.rb

```diff
@@ -1,7 +1,7 @@
 require 'alchemy/routing_constraints'
 
 Alchemy::Engine.routes.draw do
-  root :to => 'pages#show'
+  root :to => 'pages#index'
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)